README.md

```diff
@@ -104,6 +104,11 @@ from pinecone_text.sparse import SpladeEncoder
 # Initialize Splade
 splade = SpladeEncoder()
 
+# Or with "cuda" in case it's avialable
+# import pytorch
+# device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
This code should simply happen inside `SpladeEncoder.__init__()`, the user doesn't need to explicitly pass the device.

Reply:
but what if he have cuda, but don't want to use it? for example, maybe for queries you don't want to

Reply:
Anyway, I don't want to break API right now

Reply:
I'll resolve this to not let this PR hang too much

Reply:
done